bucket/lessmsi.json

```
@@ -1,11 +1,17 @@
 {
-    "homepage": "https://github.com/activescott/lessmsi/",
+    "homepage": "https://lessmsi.activescott.com/",
```

No need to keep trailing slash

```suggestion
    "homepage": "https://lessmsi.activescott.com",
```